docusaurus/docs/learn/quickstarts/browzer/index.md

```diff
@@ -0,0 +1,201 @@
+token which has an `email` claim with the value `ziggy@openziti.io` and names the identity `openziti_ziggy`:
+```bash
+identity_name="openziti_ziggy"
+auth_policy="__auth_policy_id_from_above__"
+external_id="ziggy@openziti.io"
+ziti edge create identity user "${identity_name}" \
+  --auth-policy ${auth_policy} \
+  --external-id "${external_id}" \
+  -a docker.whale.dialers
+```
+
+## Running the Ziti BrowZer Bootstrapper
+
+Running the Ziti BrowZer Bootstrapper can be done directly using NodeJS or with a container engine such as docker. Either
+way you choose to run it, you will need to establish some environment variables. If you are using NodeJS, you'll `export`
+them. For Docker-based running, you can eitehr reference the environment variables (shown below) or use a `.env` file
+if you prefer.
+
+* `NODE_ENV`: controls if the environment is `production` or `development`
+* `ZITI_BROWZER_RUNTIME_LOGLEVEL`: the log level for the Ziti BrowZer Runtime (ZBR) to use. `trace|debug|info|warn|error`
+* `ZITI_BROWZER_RUNTIME_HOTKEY`: the hotkey to activate the BrowZer settings dialog modal. default: alt+F12
+* `ZITI_CONTROLLER_HOST`: the "alternative" address for the OpenZiti controller. example: `ctrl.openziti.io`
+* `ZITI_CONTROLLER_PORT`: the port to find the OpenZiti controller at. example: `8441`
+* `ZITI_AGENT_LOGLEVEL`: the log level for the ziti-http-agent to log at. `trace|debug|info|warn|error`
+* `ZITI_AGENT_HOST`: the address the ziti-http-agent is available at. example: `browzer.openziti.io`
+* `ZITI_AGENT_LISTEN_PORT`: the port the ziti-http-agent is available at. example: `443`
+* `ZITI_AGENT_SCHEME`: the scheme to use to access the ziti-http-agent. `http|https` (https by default)
+* `ZITI_AGENT_CERTIFICATE_PATH`: the path to the certificate the ziti-http-agent presents to clients
+* `ZITI_AGENT_KEY_PATH`: the associated key for the ZITI_AGENT_CERTIFICATE_PATH
+* `ZITI_AGENT_TARGETS`: A json block representing the services to enable BrowZer for. __more on this below__
+
+```bash
+      NODE_ENV: production
+      ZITI_BROWZER_RUNTIME_LOGLEVEL: debug
+      ZITI_BROWZER_RUNTIME_HOTKEY: alt+F12
+      ZITI_CONTROLLER_HOST: ${ZITI_CTRL_EDGE_ALT_ADVERTISED_ADDRESS}
+      ZITI_CONTROLLER_PORT: ${ZITI_CTRL_EDGE_ADVERTISED_PORT}
+      ZITI_AGENT_LOGLEVEL: debug
+      ZITI_AGENT_HOST: ${ZITI_BROWZER_HTTP_AGENT_ADDRESS}
+      ZITI_AGENT_LISTEN_PORT: ${ZITI_AGENT_LISTEN_PORT}
+      ZITI_AGENT_SCHEME: https
+      ZITI_AGENT_CERTIFICATE_PATH: /etc/letsencrypt/live/your.fqdn.here/fullchain.pem
+      ZITI_AGENT_KEY_PATH: /etc/letsencrypt/live/your.fqdn.here/privkey.pem
+      ZITI_AGENT_TARGETS: __more on this below__
+```
+
+Set these values accordingly. Also shown is the reuse of certificates provisioned in this case by LetsEncrypt. BrowZer
+requires a wildcard certificate to be provisioned that aligns to the `ZITI_AGENT_HOST` value used. For example, if you had
```

Comment:
At this time, It's required to be a wildcard certificate so that you can use more than one service. If you _ONLY_ have one service, you don't need a wildcard cert but for now, I'm not going to get into these sorts of questions/alternatives in the doc